generators/client/templates/angular/src/main/webapp/app/admin/user-management/_user-modal.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+import { NgbModal, NgbModalRef } from '@ng-bootstrap/ng-bootstrap';
+
+import { UserMgmtDialogComponent } from './user-management-dialog.component';
+import { User } from './user.model';
+import { UserService } from './user.service';
+
+@Injectable()
+export class UserModalService {
+    private isOpen = false;
```

Comment:
we can have it one per entity ... ( considering what we currently did for the other health / login ... )